server/index.js

```diff
@@ -1,8 +1,19 @@
 require('dotenv').config()
 const express = require('express');
 const app = express();
+const multer = require('multer');
+const { handleUpload } = require('./controllers/index');
 
 app.use(express.static('./client/dist'));
+const upload = multer();
```

Comment:
Maybe move this line outside of the middleware functions. It's an odd place for it